src/main/java/com/github/bot/curiosone/core/extraction/BadWords.java

```diff
@@ -59,9 +67,11 @@ private static void loadSentences() {
   }
 
   /**
-   * Check if the phrase contains one of the known bad words.
-   * @param phrase to check
-   * @return answer if the input is known
+   * Returns an answer for the given phrase.
+   * @param phrase the original phrase to be answered
+   * @return an Optional instance.
```

Comment:
Optional instance???? The doc is also wrongly aligned.
Make it more meaningful.